tools/jslint.js

```diff
@@ -13,9 +13,15 @@ const CLIEngine = require('./eslint').CLIEngine;
 const glob = require('./eslint/node_modules/glob');
 
 const cwd = process.cwd();
-const cli = new CLIEngine({
+const cliOptions = {
   rulePaths: rulesDirs
-});
+};
+
+// Check if we should fix errors that are fixable
+if (process.argv.indexOf('-F') !== -1)
```

Comment:
@silverwind wrote:

> process.argv.includes('-F')?

I don't feel strongly about it, but I went with `indexOf() !== -1` over `includes()` so that this could be used in the LTS branch and to be consistent with line 47 and elsewhere in the file that was already using `indexOf() !== -1` for existence.